optimize/components/userguide/processes.md

```diff
@@ -7,7 +7,7 @@ description: "The processes section gives you an overview of all the processes a
 ## Overview
 
 The processes section gives you an overview of all the processes and their status on a single page.
-This section also allows you to set a process owner and take responsibility of a process, viewing time, and quality KPIs to track process performance. Additionally, a **Management Dashboard** can be found at the top of the page, which integrates data from all processes in one view. The optional date filter is applied to all the reports shown in this view.
+This section also allows you to set a process owner and take responsibility of a process, viewing time, and quality KPIs to track process performance. Additionally, a **Adoption Dashboard** can be found at the top of the page, which integrates data from all processes in one view. The optional date filter is applied to all the reports shown in this view.
 ![Processes page](./img/processOverview.png)
```

Comment:
@christinaausley, seeing as this was renamed, can you check if other areas in the Optimize docs reference the "Management Dashboard?" 

Also, please add a keyword to the page metadata to include "management dashboard" so it still shows up in search. [Here](https://github.com/camunda/camunda-platform-docs/blob/d0265635a3c43dde0087be2647e5b3ec087ae37e/docs/guides/automating-a-process-using-bpmn.md?plain=1#L5) is an example. Do you remember if keywords can have spaces or need dashes?

Reply:
Yes spaces are fine!

Reply:
Can confirm `processes.md` is the only page referencing "Management Dashboard".